ApiClient/Source/Query.swift

```diff
@@ -29,10 +29,11 @@ import Alamofire
 protocol Query {
     associatedtype Result
     
-    var resource: String { get }
+    var url: URL { get }
+    var host: String { get }
```

Comment:
why does the protocol still need the host when we have a url?

Reply:
not needed, removing.